### ISSUE TITLE
module 'freud.diffraction' has no attribute 'StaticStructureFactorDirect'

### ISSUE THREAD

**Issue by Roy-Kid** [glotzerlab/freud#898]
**Describe the bug**
I am trying to follow the manual to use `StaticStructureFactorDirect`. When I use
```
import freud
sf = freud.diffraction.StaticStructureFactorDirect(
```
it tells me `module 'freud.diffraction' has no attribute 'StaticStructureFactorDirect'`, and if 

```
from freud.diffraction import StaticStructureFactorDebye
```
only *Debye* style can import
![image](https://user-images.githubusercontent.com/42854324/148156320-b0bd79e8-adfe-4593-b827-0ede77c495b8.png)

could you tell me how to calculate Sq directly?
Thanks in advanced!

**Comment by DomFijan** [comment]
Have you installed freud from pypi (pip) or conda? If so the latest release (2.7.) on pypi and conda does not have the direct method included yet. If you would like to use the direct method, you will have to install freud by cloning the repo (or pulling the latest) and manually installing freud (you can find more detailed instructions for this [here](https://freud.readthedocs.io/en/latest/gettingstarted/installation.html) under "Compile from source" ).

**Comment by Roy-Kid** [comment]
Yes, I do use pip and conda to install freud. I will compile it from source. Thanks for your kindly help! BTW, why Direct style method can not be used by pip and conda, will it be removed or something?

**Comment by DomFijan** [comment]
The pull request that implements the direct method was merged after 2.7. was released. It should be available through pip/conda on next release.
